docs/getting-started/cases-req.asciidoc

```diff
@@ -34,7 +39,12 @@ a|
 * **All** for the *Cases* feature under *Security*
 * **All** for the *{connectors-feature}* feature under *Management*
 
-NOTE: Roles without **All** *{connectors-feature}* feature privileges cannot create, add, delete, or modify case connectors.
+[NOTE]
+====
+Roles without **All** *{connectors-feature}* feature privileges cannot create, add, delete, or modify case connectors.
+
+By default, `All` for the *Cases* feature includes authority to delete cases, delete alerts and comments from cases, and edit case settings unless you customize the sub-feature privileges.
```

Comment:
It might make sense to switch this with the previous sentence about roles without actions and connector privs. That way the note lines up with the order of the bullets. Also made a minor edit to break up the sentence a bit.

```suggestion
By default, **All** for the *Cases* feature allows you to delete cases, delete alerts and comments from cases, and edit case settings. Customize the sub-feature privileges to limit feature access.  
```

Reply:
Agree with Nastasha's comment, just a tiny tweak to the suggestion, to make it clearer that this action is optional.
```suggestion
By default, **All** for the *Cases* feature allows you to delete cases, delete alerts and comments from cases, and edit case settings. You can customize the sub-feature privileges to limit feature access.
```